runPerftool.psm1

```diff
@@ -45,13 +45,21 @@ $ScriptBlockEnableToolPermissions = {
 
 $ScriptBlockCleanupFirewallRules = {
     param($port, $creds)
-    Write-Output $creds.GetNetworkCredential().Password | sudo -S ufw delete allow $port | Out-Null
-} # $ScriptBlockCleanupFirewallRules()
+    if ([String]::IsNullOrWhiteSpace($creds.GetNetworkCredential().Password)) {
```

Comment:
could creds be $null ? We should consider adding a validation script to the process*commands functions

Reply:
$creds is never $null because I think we always recreate an empty creds object [Parameter(Mandatory=$False)] [PSCredential] $SendComputerCreds = [System.Management.Automation.PSCredential]::Empty